misc/msvs/vcxproj.template

```diff
@@ -10,16 +10,18 @@
     <VCProjectUpgraderObjectName>NoUpgrade</VCProjectUpgraderObjectName>
   </PropertyGroup>
   %%PROPERTIES%%
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+  <PropertyGroup Label="Configuration">
+    <PlatformToolset>%%PlatformToolset%%</PlatformToolset>
+  </PropertyGroup>
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" Condition="'$(PlatformToolset)' != 'Clang'" />
```

Comment:
You can't do this, this breaks building in VS with clang, which is a perfectly valid way of building things, VS supports both platform toolsets. The assumption that if the platform toolset is clang means that we're on Rider is not correct. If you need to detect whether you're in VS or Rider, you need to do something different. Maybe see if Rider sets some property that you can detect here for this condition?